patterns/java/src/refactoring_guru/patterns/singleton/example/non_therad_safe/Client.java

```diff
@@ -0,0 +1,9 @@
+package refactoring_guru.patterns.singleton.example.non_therad_safe;
```

Comment:
I think this can be merged to a Demo class (same with thread safe example). From my earlier comment: "[have] a client (or demo) class". You have a demo, so client is useless. Please, comment if you think different.